ipxe/pxeboot.conf

```diff
@@ -0,0 +1,38 @@
+# dnsmasq netboot config file
+
+enable-tftp
+tftp-root=/srv/tftpboot
+
+# prevent some dnsmsq magic that will confuse old PXE clients
+dhcp-no-override
+
+# set next-server
+dhcp-option=option:tftp-server,"10.150.1.11"
+
+# set tags according to client architecture option received in DISCOVER
+# see https://www.rfc-editor.org/rfc/rfc4578#section-2.1 for an incomplete list
+dhcp-match=set:efi-x86_64-http,option:client-arch,16
+dhcp-match=set:efi-x86_64,option:client-arch,9
+dhcp-match=set:efi-x86_64,option:client-arch,7
+dhcp-match=set:efi-x86,option:client-arch,6
+dhcp-match=set:bios,option:client-arch,0
+
+# BIOS PXE tftp boot
+dhcp-option=tag:bios,option:bootfile-name,ipxe/ipxe.pxe
+
+# UEFI tftp boot
+dhcp-option=tag:efi-x86_64,option:bootfile-name,ipxe/ipxe.efi
+
+# UEFI http boot
+dhcp-option=tag:efi-x86_64-http,option:bootfile-name,http://10.150.1.11/tftpboot/ipxe/ipxe.efi
+dhcp-option=tag:efi-x86_64-http,option:vendor-class,HTTPClient
+
+# UEFI ipxe binary from Arch Linux' ipxe package, useful for testing
+#dhcp-boot=tag:efi-x86_64,arch-netboot/ipxe-arch.16e24bec1a7c.efi
+
+# UEFI ipxe binnary built that loads commands from http://martchus.no-ip.biz/ipxe/leap-15.4 to install Leap 15.4 with autoyast
+# note: This image has been used to install Leap 15.4 on ow19/ow20, see https://progress.opensuse.org/issues/111473#note-44 for
+#       details.
+#dhcp-boot=tag:efi-x86_64,ipxe-own-build/ipxe.efi
```

Comment:
I could live with having this disabled code from my previous experiments and setup work removed. So we should likely not add them to Git.